package-rust.py

```diff
@@ -133,6 +135,8 @@ def run(args):
     rustc_dir + "/COPYRIGHT " + rustc_dir + "/LICENSE-APACHE " + rustc_dir + "/LICENSE-MIT " + \
     "> " + license_file
 run(["sh", "-c", cmd])
+license_rtf = TEMP_DIR + "/LICENSE.rtf"
+run(["pandoc", "-s", license_file, "-o", license_rtf])
```

Comment:
If we don't care about fancy formatting, I think it should be possible to convert license to a plain-text looking RTF with a bunch of regex substitutions.